pkg/sfu/receiver.go

```diff
@@ -841,6 +844,11 @@ func (w *WebRTCReceiver) GetMonotonicNowUnixNano() int64 {
 	return w.baseTime.Add(time.Since(w.baseTime)).UnixNano()
 }
 
+func (w *WebRTCReceiver) AddOnReady(fn func()) {
+	// webRTCReceiver is always ready after created
+	fn()
```

Comment:
Are there any locking issues here? What if caller is holding some lock and the callback also needs that lock?

Reply:
It is ok that the downtrack doesn't hold lock when calling `AddOnReady`, will add a comment for future that the callback will be called immediately if receiver is already ready.